lib/rabbit/command/rabbit-slide.rb

```diff
@@ -705,6 +707,7 @@ def generate_readme
 
       def readme_content
         markup_language = @data.markup_language
+        markup_language = :markdown if markup_language == :pdf
```

Comment:
Hmm. Let's add `--readme-markup-language` instead of reusing `--markup-language`. (We can use `--markup-language` value as the default value of `--readme-markup-language`.)

Reply:
Maybe we should consider this after #183?